com.github.reds.LogisimEvolution.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.reds.LogisimEvolution
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+  
+command: LogisimEvolution.sh
+finish-args:
+  - --socket=fallback-x11
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+  - --env=_JAVA_AWT_WM_NONREPARENTING=1
+  - --filesystem=xdg-documents
+  - --persist=.java
```

Comment:
This one is odd. Does it really use `.java` for something?

Reply:
There's `.java/.userPrefs` for saving preferences and first-run logic, and also `.java/fonts` for some Java fontconfig cache.